pinot-common/src/main/java/org/apache/pinot/common/assignment/InstancePartitionsUtils.java

```diff
@@ -93,6 +94,11 @@ public static InstancePartitions fetchInstancePartitions(HelixPropertyStore<ZNRe
     return znRecord != null ? InstancePartitions.fromZNRecord(znRecord) : null;
   }
 
+  public static String getInstancePartitonNameForTier(String tableName, String tierName) {
```

Comment:
nit: the naming convention seems getInstancePartiton`s`NameForTier

Reply:
Ack